test/Core.Test/OrganizationFeatures/OrganizationSubscriptionUpdate/UpdateSecretsManagerSubscriptionCommandTests.cs

```diff
@@ -0,0 +1,631 @@
+﻿using Bit.Core.Entities;
+using Bit.Core.Enums;
+using Bit.Core.Exceptions;
+using Bit.Core.Models.Business;
+using Bit.Core.OrganizationFeatures.OrganizationSubscriptionUpdate;
+using Bit.Core.Repositories;
+using Bit.Core.Services;
+using Bit.Core.Tools.Enums;
+using Bit.Core.Tools.Models.Business;
+using Bit.Core.Tools.Services;
+using Bit.Core.Utilities;
+using Bit.Test.Common.AutoFixture;
+using Bit.Test.Common.AutoFixture.Attributes;
+using NSubstitute;
+using Xunit;
+
+namespace Bit.Core.Test.OrganizationFeatures.OrganizationSubscriptionUpdate;
+
+[SutProviderCustomize]
+public class UpdateSecretsManagerSubscriptionCommandTests
+{
+    [Theory]
+    [BitAutoData]
+    public async Task UpdateSecretsManagerSubscription_NoOrganization_Throws(
```

Comment:
On all these negative tests, please verify that none of the final dependencies are called, e.g. database writes, StripePaymentService, IMailService. Maybe put those assertions in a private method to avoid repetition.

Reply:
resolve